pkg/value/value.go

```diff
@@ -32,3 +32,19 @@ const (
 func IsStaleNaN(v float64) bool {
 	return math.Float64bits(v) == StaleNaN
 }
+
+// MinInt64 returns the minimum value between two int64 values.
+func MinInt64(a, b int64) int64 {
```

Comment:
in `tsdb/wal/live_reader.go`  you can remove the duplicate func and use thie one instead.

`func min(i, j int) int {`